tests/testthat/test-utilities-individual.R

```diff
@@ -50,3 +50,30 @@ test_that("It create IndividualCharacteristics when numerical values are empty",
   expect_equal(individualCharacteristics$population, "European_ICRP_2002")
   expect_equal(individualCharacteristics$gender, "MALE")
 })
+
+## context("writeIndividualToXLS")
+
+test_that("`writeIndividualToXLS()` writes correct data to a spreadsheet", {
+  skip_if_not_installed("withr")
```

Comment:
What would happen if `withr` is not installed and `skip_if_not_installed` is not defined? Would the test fail? In this case, I would prefer it to fail. If someone is developing the package, he should also run the tests.